optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/persistence/util/VehicleRoutingDistanceTypeComparison.java

```diff
@@ -36,7 +36,7 @@
 
 public class VehicleRoutingDistanceTypeComparison extends LoggingMain {
 
-    private final ScoreManager<VehicleRoutingSolution> scoreManager;
+    private final ScoreManager<VehicleRoutingSolution, ?> scoreManager;
```

Comment:
We want to use the actual score types in these cases, which can simply be looked up by clicking on VehicleRoutingSolution